### PR TITLE
updated TP to use API 1.5 and TO perl to use 1.5

### PR DESCRIPTION
**Pull request by mattjackson220**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any Issue 

## Which Traffic Control components are affected by this PR?

- Traffic Ops
- Traffic Portal

## What is the best way to verify this PR?

Verify that requests from TP go to the 1.5 API and that TO perl accepts requests to the 1.5 API

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_ops/app/lib/TrafficOpsRoutes.pm [hunk 1]
```diff
@@ -64,6 +64,12 @@ sub define {
 	# Traffic Stats Extension 1.4
 	$self->traffic_stats_routes( $r, $version );
 
+	# 1.5 Routes
```
Comment (mitchell852) [inline traffic_ops/app/lib/TrafficOpsRoutes.pm]: the formatting police are going to get you :)

Comment (rawlinp) [reply traffic_ops/app/lib/TrafficOpsRoutes.pm]: 🚨 wee woo 🚨

**Comment by mitchell852** [comment]
Pretty sure this PR can NOT be merged until #4198 is merged. #4198 is the PR that introduces api 1.5 and is necessary before you can make calls to 1.5.

File: traffic_ops/app/lib/TrafficOpsRoutes.pm (hunk 1 above)
Comment (mattjackson220) [reply traffic_ops/app/lib/TrafficOpsRoutes.pm]: fixed. sorry about that

**Comment by ocket8888** [comment]
IDK if that's necessarily true, TO (go) already serves 1.5 and making TO (perl) do it rounds out that support, so TP should be fine. Anything else is pinned to <= 1.4 so it won't care.

**Comment by rawlinp** [comment]
> IDK if that's necessarily true, TO (go) already serves 1.5 and making TO (perl) do it rounds out that support, so TP should be fine. Anything else is pinned to <= 1.4 so it won't care.

TO-Go doesn't yet serve any 1.5 routes yet

**Comment by ocket8888** [comment]
> _"TO-Go doesn't yet serve any 1.5 routes yet"_
That's right, idk what's wrong with me.

**Pull request merged**